OpenTrons/opentrons-api
# fix(app, components): update style of LabwareStackModal

# Overview

Updates style of `LabwareStackModal` component according to design changes

Closes RQA-2971

## Test Plan and Hands on Testing

- Begin protocol setup for a protocol with stacked entities ([example](https://github.com/user-attachments/files/16590367/demo.py.zip))
- On desktop, proceed to Labware setup > map view
- Click labware stacks and verify that if there is only a single labware rendered, it is not blue highlighted
- Verify that right hand elements (images/renders) are aligned center
- Repeat above for ODD and verify that modal is not scrollable

## Changelog

- remove highlight from labware if it is not part of a stack
- align all right side content (images/labware renders) to center
- update HeaterShaker image
- fix scrolling issue on ODD

## Review requests

see test plan

## Risk assessment

low
